docs/manual/prophet-objects.rst

```diff
@@ -119,6 +95,10 @@ will use to determine the type of the stub:
         }
     }
 
+**phpspec** 2.* supports the use of `@param` annotations instead of parametric
```

Comment:
actually, not if it's going to trigger a big consistency change across the docs ;-)